core/src/main/java/bio/terra/pearl/core/service/datarepo/DataRepoExportService.java

```diff
@@ -132,21 +113,12 @@ public void createDataset(StudyEnvironment studyEnv, String datasetName) {
         dataRepoJobService.create(job);
     }
 
-    public void ingestDatasets() {
-        List<Dataset> outdatedDatasets = datasetDao.findAll().stream().filter(dataset -> dataset.getLastExported().isBefore(Instant.now().minus(1, ChronoUnit.MINUTES))).toList();
-
-        logger.info("Found {} study environments requiring dataset ingest", outdatedDatasets.size());
-
-        for(Dataset dataset : outdatedDatasets) {
-            logger.info("Ingesting data for study environment ID {}", dataset.getStudyEnvironmentId());
-            ingestDataForStudyEnvironment(dataset);
-        }
-    }
-
-    public String uploadCsvToAzureStorage(UUID studyEnvironmentId) {
+    public String uploadCsvToAzureStorage(UUID studyEnvironmentId, String datasetName) {
         ExportOptions exportOptions = new ExportOptions(false, false, false, ExportFileFormat.TSV, null);
 
-        String blobName = studyEnvironmentId + "_" + Instant.now() + ".csv";
+        //Even though this is actually formatted as a TSV, TDR only accepts files ending in .csv or .json.
+        //In the DataRepoClient call, we specify that the CSV delimiter is "\t", which will make it all work fine.
```

Comment:
Oh, I see. I hope there aren't corner cases that this introduces. Maybe tab-delimited CSVs are generally well-supported and accepted.

The nice thing about CSV is that cells can contain anything, including tabs and newlines. If a cell contains anything that would otherwise confuse the parser (like `,` or `\n`) then it gets double-quoted (with any embedded double-quotes escaped)... though I guess `\t` is just as good as `,` as long as the parser adapts and correctly implements the rules in the same way.

And as long as we're correctly implementing those rules when generating the input file. I remember a time when TSV generation libraries didn't deal well with embedded tabs, but maybe things have evolved since then.